website/src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -37,13 +37,13 @@ const LanguageSelector = () => {
       await router.push(path, path, { locale });
       router.reload();
     },
-    [router]
+    [router, setCookie]
   );
 
   const { language: currentLanguage } = i18n;
   return (
     <Select onChange={languageChanged} defaultValue={currentLanguage}>
-      {localesAndNames.map(({ locale, name }) => (
+      {localesAndNames?.map(({ locale, name }) => (
```

Comment:
the previous `useMemo` should return an empty array in case there are no locales.

Reply:
Storybook throws an error `Cannot read properties of undefined (reading 'map')`.  Adding this fixes that.